app/bower_components/angular-idle-service/src/IdleStorage/idleStorage.ts

```
@@ -0,0 +1,76 @@
+module IdlePackage.Modules {
```

Biome (1.9.4)

[error] Use the namespace keyword instead of the outdated module keyword.

The module keyword is deprecated to avoid any confusion with the ECMAScript modules which are often called modules.
Safe fix: Use namespace instead.


(lint/suspicious/useNamespaceKeyword)